test/verify/check-packagekit

```diff
@@ -190,8 +190,8 @@ class TestUpdates(NoSubManCase):
         self.assertEqual(b.text("#available-updates button#install-all"), "Install all updates")
         b.click("#available-updates button#install-all")
 
-        b.wait_in_text("#state", "Applying updates")
-        b.wait_visible("#app div.progress-bar")
+        # applying updates panel present
```

Comment:
Please squash the test adjustments into the commits that change the code. Otherwise this breaks rebasing.